code/modules/experisci/experiment/experiments.dm

```diff
@@ -0,0 +1,139 @@
+/datum/experiment/scanning/random/slime
+	name = "Base Slime Experiment"
+	total_requirement = 10
+	max_requirement_per_type = 4
+
+/datum/experiment/scanning/random/slime/calibration
+	name = "Slime Sample Test"
+	description = "Lets see if our scanners can pick up the genetic data from a simple slime extract."
+	required_atoms = list(/obj/item/slime_extract/grey = 1)
+
+/datum/experiment/scanning/random/slime/easy
+	name = "Easy Slime Survey"
+	description = "A wealthy client has requested that we provide samples of data from several basic slime cores."
+	possible_types = list(/obj/item/slime_extract/orange,
+							/obj/item/slime_extract/purple,
+							/obj/item/slime_extract/blue,
+							/obj/item/slime_extract/metal,
+							/obj/item/slime_extract/yellow,
+							/obj/item/slime_extract/darkpurple,
+							/obj/item/slime_extract/darkblue,
+							/obj/item/slime_extract/silver)
+
+/datum/experiment/scanning/random/slime/moderate
+	name = "Moderate Slime Survey"
+	description = "Central Command has asked that you collect data from several common slime cores."
+	possible_types = list(/obj/item/slime_extract/bluespace,
+							/obj/item/slime_extract/sepia,
+							/obj/item/slime_extract/cerulean,
+							/obj/item/slime_extract/pyrite,
+							/obj/item/slime_extract/red,
+							/obj/item/slime_extract/green,
+							/obj/item/slime_extract/pink,
+							/obj/item/slime_extract/gold)
+
+/datum/experiment/scanning/random/slime/hard
+	name = "Challenging Slime Survey"
+	description = "Another station has challenged your research team to collect several challenging slime cores, \
+	 				including a very valuable rainbow core. Are you up to the task?"
+	required_atoms = list(/obj/item/slime_extract/rainbow = 1)
```

Comment:
Okay i get you want to make this hard, but this may be a bit too much if this locks techs behind it.